renku/data/shacl_shape.json

```diff
@@ -529,6 +529,13 @@
                "datatype": {
                   "@id": "xsd:string"
                }
+            },
+            {
+               "nodeKind": "sh:Literal",
+               "path": "sdsc:external",
```

Comment:
We should really add this to the `renku:` ontology in https://github.com/SwissDataScienceCenter/renku-ontology/tree/gh-pages

Reply:
I've already fixed this, I guess you looked at an older commit.